components/Helpers/samples/ThemeListener.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Theme Listener
+author: williamabradley
+description: The ThemeListener allows you to keep track of changes to the System Theme.
+keywords: Helpers, Theming, theme listerner, themes
+dev_langs:
+  - csharp
+category: Helpers
+subcategory: Layout
+discussion-id: 0
+issue-id: 0
+---
+
+# Theme Listener
+
+The [Theme Listener](/dotnet/api/microsoft.toolkit.uwp.ui.helpers.themelistener) class allows you to determine the current Application Theme, and when it is changed via System Theme changes.
+
+KNOWN ISSUE: `ThemeListener` might not work in WinUI3 applications.
+
+> [!Sample ThemeListenerSample]
+
+```csharp
```

Comment:
We should have a bit more around these two code blocks. There's no labels or indicators visible to the end user, beyond the code itself.

Or maybe since we have the sample showing basic usage, we just remove these code blocks? Keeps it consistent across all samples.

![image](https://user-images.githubusercontent.com/9384894/235787267-76597815-39de-41ee-a0e4-b580f1ba2483.png)

Reply:
Agreed! We need to figure out at some point what we want to show where in terms of code snippets vs. samples. (Same for listing e.g. properties).

Removed the code for now as the sample is the same 👍.